README.md

```diff
@@ -1,23 +1,25 @@
-# Jujutsu VCS
+<div class="title-block" style="text-align: center;" align="center">
 
-![](https://img.shields.io/github/license/martinvonz/jj) ![](https://img.shields.io/github/v/release/martinvonz/jj) ![](https://img.shields.io/github/release-date/martinvonz/jj) ![](https://img.shields.io/crates/v/jj-cli)
+# Jujutsu—a version control system
+
+![](https://img.shields.io/github/license/martinvonz/jj)
+![](https://img.shields.io/github/v/release/martinvonz/jj)
+![](https://img.shields.io/github/release-date/martinvonz/jj)
+![](https://img.shields.io/crates/v/jj-cli)
 <br/>
-![](https://github.com/martinvonz/jj/workflows/build/badge.svg) ![](https://img.shields.io/codefactor/grade/github/martinvonz/jj/main) ![](https://img.shields.io/librariesio/github/martinvonz/jj)
+![](https://github.com/martinvonz/jj/workflows/build/badge.svg)
+![](https://img.shields.io/codefactor/grade/github/martinvonz/jj/main)
+![](https://img.shields.io/librariesio/github/martinvonz/jj)
 
-- [Disclaimer](#disclaimer)
-- [Introduction](#introduction)
-- [Getting started](#getting-started)
-- [Status](#status)
-- [Related work](#related-work)
+**[Homepage] &nbsp;&nbsp;&bull;&nbsp;&nbsp;**
+**[Installation] &nbsp;&nbsp;&bull;&nbsp;&nbsp;**
+**[Getting Started Tutorial]**
 
-## Disclaimer
+[Homepage]: https://martinvonz.github.io/jj
+[Installation]: https://martinvonz.github.io/jj/latest/install-and-setup.html
```

Comment:
You should change this to <https://martinvonz.github.io/jj/latest/install-and-setup>. The link you have will (unfortunately) no longer be valid shortly (after the release) because of https://github.com/martinvonz/jj/pull/2317.